applications/serial_lte_modem/doc/slm_testing.rst

```diff
@@ -563,17 +563,19 @@ You must register the same PSK and PSK identity on the server side.
 TCP server
 ==========
 
-.. |global_private_address| replace:: the nRF91 Series DK must have a global private address.
-   The radio network must be configured to route incoming IP packets to the nRF91 Series DK.
-
-.. |global_private_address_check| replace::    To check if the setup is correct, use the ``AT+CGDCONT?`` command to check if the local IP address allocated by the network is a reserved private address of class A, B, or C (see `Private addresses`_).
-   If it is not, ping your nRF91 Series DK from the destination server.
+.. |global_private_address| replace:: the nRF91 Series DK must have a public IP address and the radio network must be configured to route incoming IP packets to the nRF91 Series DK.
```

Comment:
I'm not sure what "global private address" is meant to mean but definitely it is not a public IP address.

So this sentence is now wrong.

Reply:
I agree that the tag is wrong and should be updated, but the sentence reflects what we want. 

I don't think that global private address actually means anything in relation to IP addresses.

Reply:
I went with the minimum change required, but for future work, it does make sense to update that tag to reflect the current content since it is not used outside of the page.